src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -324,12 +341,13 @@ const General = ({
           </Select>
         </Item>
 
+        {childProfiles.rf.length > 0 && <Item label="RF Profile">{childProfiles.rf[0]?.name}</Item>}
```

Comment:
Change to just `<Item label="RF Profile">{childProfiles.rf?.[0]?.name || 'N/A'}</Item>`